resource_estimator/src/estimates.rs

```diff
@@ -79,21 +82,28 @@ fn estimate_single(
     // create error budget partitioning
     let partitioning = job_params.error_budget().partitioning(&logical_resources)?;
 
-    let mut estimation =
-        PhysicalResourceEstimation::new(ftp, qubit, logical_resources, partitioning);
+    let mut estimation = PhysicalResourceEstimation::new(
+        ftp,
+        qubit,
+        TFactoryBuilder::default(),
+        logical_resources,
+        partitioning,
+    );
     if let Some(logical_depth_factor) = job_params.constraints().logical_depth_factor {
         estimation.set_logical_depth_factor(logical_depth_factor);
     }
     if let Some(max_t_factories) = job_params.constraints().max_t_factories {
-        estimation.set_max_t_factories(max_t_factories);
+        estimation.set_max_factories(max_t_factories);
```

Comment:
to confirm that this is itended

Reply:
Yes, the estimator is generic over `FactoryBuilder` which just considers that there are some magic states. The system is based on T states.

Reply:
I am seeing changes in naming. I am not against them but would like:
- them to be consistent
- understand what would be the standard naming

There are two t_factories on lines 97-98 and one factory without 't'.

There are magic states in code below instead of t-states. If we want to start a renaming, let us agree and apply it everywhere.

Reply:
You're right. I missed quite some instances for renaming and will fix this soon.

I suggest to use `t_states` in the specific system-related code, and `magic_states` in the generic estimator-related code.